src/caf/toolkit/translation.py

```diff
@@ -707,6 +767,42 @@ def pandas_matrix_zone_translation(
     )
 
 
+@overload
+def pandas_vector_zone_translation(
+    vector: pd.Series,
+    translation: pd.DataFrame,
+    translation_from_col: str,
+    translation_to_col: str,
+    translation_factors_col: str,
+    from_unique_index: list[Any],
+    to_unique_index: list[Any],
+    translation_dtype: Optional[np.dtype] = None,
+    vector_infill: float = 0.0,
+    translate_infill: float = 0.0,
+    check_totals: bool = True,
+) -> pd.Series:
+    # pylint: disable=too-many-arguments
+    ...  # pragma: no cover
+
+
+@overload
+def pandas_vector_zone_translation(
+    vector: pd.DataFrame,
+    translation: pd.DataFrame,
+    translation_from_col: str,
+    translation_to_col: str,
+    translation_factors_col: str,
+    from_unique_index: list[Any],
+    to_unique_index: list[Any],
+    translation_dtype: Optional[np.dtype] = None,
+    vector_infill: float = 0.0,
+    translate_infill: float = 0.0,
+    check_totals: bool = True,
+) -> pd.DataFrame:
+    # pylint: disable=too-many-arguments
+    ...  # pragma: no cover
```

Comment:
This second overload is currently causing the following error with MyPy linting:
![image](https://github.com/Transport-for-the-North/caf.toolkit/assets/68055450/bd89ab21-f3e0-48b6-b41a-318cf9aac6a0)

This happens because we don't have the pandas-stubs installed when performing mypy checks, meaning all pandas type-hints are assumed to be `Any`. 

Best solution is to install pandas-stubs to remove this issue. However, this introduces a number of other errors we have overlooked until now. I've open a new PR (#78) that deals with this.